adoptingerlang.org

```diff
@@ -3928,25 +3947,34 @@ Each init container is run in order to completion before the next is started, wh
 
 *** Tilt for Local Development
 
-[[https://tilt.dev/][Tilt]] is a tool for deploying and updating Docker images and Kubernetes deploys locally for development purposes. By default it will only build and deploy services against a local Kubernetes, such as =microk8s=, =minikube=, etc. to safe guard against accidentally sending your development environment into production!
+[[https://tilt.dev/][Tilt]] is a tool for deploying and updating Docker images and Kubernetes deploys
+locally for development purposes. By default it will only build and deploy
+services against a local Kubernetes, such as =kind=, =microk8s=, etc. to safe guard against accidentally sending your development environment into production!
 
-The easiest way to get started with Tilt is using a registry within the local Kubernetes cluster. For =microk8s= a registry can be enabled easily, we will need DNS as well so enable that at this time as well:
+The easiest way to get started with Tilt is using a registry within the local
```

Comment:
I think it'd make sense to at least explain a registry for what, or put a link to the registry to explain it's for the docker images.

Reply:
good point.